roles/telemetry_logging/tasks/logging_journal_tests.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+- name: Get journals
+  ansible.builtin.shell:
+    cmd:
+      tstamp=$(date -d '30 minute ago' "+%Y-%m-%d %H:%M:%S") 
+      journalctl -t "{{ item }}" --no-pager -S "${tstamp}" | wc -l
+  register: journal_wc
+  changed_when: false
+  failed_when: false
```

Comment:
This pattern of using failed_when: false should be revised.

The roles in this repo are typically included by playbooks using ``ignore_errors: true``, which means that all the tasks will run, regardless of pass/fail results.

I would recommend adding the fail condition to the test task directly, instead of using a later task to check the status.

I see that you're trying to add reporting, however, we have the existing functionality to do this without adding additional tasks for each test.
The custom logger was implemented to resolve a similar issue, where we wanted to record the pass/fail status for certain tasks into a file  in order to report to polarian. Recently, Marihan has added some additional functionality to produce a test report that isn't limited to tagged tasks.